opencti-platform/opencti-front/src/private/components/settings/roles/CapabilitiesList.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FunctionComponent } from 'react';
+import { PreloadedQuery, usePreloadedQuery } from 'react-relay';
+import * as R from 'ramda';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { useFormatter } from '../../../../components/i18n';
+import { roleEditionCapabilitiesLinesSearch } from './RoleEditionCapabilities';
+import {
+  RoleEditionCapabilitiesLinesSearchQuery,
+} from './__generated__/RoleEditionCapabilitiesLinesSearchQuery.graphql';
+import { Role_role$data } from './__generated__/Role_role.graphql';
+
+interface CapabilitiesListProps {
+  queryRef: PreloadedQuery<RoleEditionCapabilitiesLinesSearchQuery>,
+  role: Role_role$data,
+}
+
+const CapabilitiesList: FunctionComponent<CapabilitiesListProps> = ({ queryRef, role }) => {
+  const { t } = useFormatter();
+  const roleCapabilities = (role.capabilities ?? []).map((n) => ({ name: n?.name })) as { name: string }[];
+  const { capabilities } = usePreloadedQuery<RoleEditionCapabilitiesLinesSearchQuery>(
+    roleEditionCapabilitiesLinesSearch,
+    queryRef,
+  );
+
+  return (
+    <List>
+      {/* eslint-disable-next-line consistent-return,array-callback-return */}
```

Comment:
if think your function should return <></> if not checked

Reply:
fixed